<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui_Settings</class>
 <widget class="QDialog" name="Gui_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/res_icons.qrc">
    <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="interfaceGroup">
         <property name="title">
          <string>Interface</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="interfaceLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="iconSizeLabel">
              <property name="text">
               <string>Toolbar icon size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="iconSize">
              <property name="minimum">
               <number>8</number>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
              <property name="singleStep">
               <number>8</number>
              </property>
              <property name="value">
               <number>48</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="restoreSize">
            <property name="text">
             <string>Restore main window size on startup</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Modules</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="extractModules">
            <property name="text">
             <string>Extract modules from packages</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filesystemGroup">
         <property name="title">
          <string>Filesystem</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="enableFsDebug">
            <property name="text">
             <string>Enable debugging</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ignoreLinks">
            <property name="text">
             <string>Ignore links</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="settingsLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonAccept">
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res_icons.qrc">
         <normaloff>:/icons/accept</normaloff>:/icons/accept</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/res_icons.qrc">
         <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/res_icons.qrc"/>
 </resources>
 <connections/>
</ui>
